101-web-app-certificate-from-key-vault/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
```

Comment:
Won't pass but in the future when we can enable CI on existing resources, it'll be one less change to make :)